internal/pkg/agent/application/upgrade/upgrade.go

```diff
@@ -220,6 +213,8 @@ func (u *Upgrader) Ack(ctx context.Context) error {
 		return err
 	}
 
+	marker.Acked = true
```

Comment:
@michel-laterman this seems to be needed here, since the action was just acked so we can persist the market with correct "acked" value. Please double check.